src/sql/src/func.rs

```diff
@@ -2489,7 +2489,7 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         },
         "min" => Aggregate {
             params!(Bool) => AggregateFunc::MinBool => Bool, oid::FUNC_MIN_BOOL_OID;
-            params!(Int16) => AggregateFunc::MinInt32 => Int16, 2133;
```

Comment:
yikes! I've thought for a long time we need a test that executes all functions with some default value for their arguments, to make sure these kinds of things can't happen.